rvps/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#![allow(clippy::new_without_default)]
+
+pub mod cache;
+pub mod extractors;
+pub mod pre_processor;
+pub mod reference_value;
+
+use std::time::SystemTime;
+
+use anyhow::{anyhow, Result};
+use chrono::{DateTime, Utc};
+use extractors::{Extractors, ExtractorsImpl};
+use pre_processor::{PreProcessor, PreProcessorAPI, Ware};
+use serde::{Deserialize, Serialize};
+
+pub use cache::Cache;
+pub use reference_value::{ReferenceValue, TrustedDigest};
+
+/// Default version of Message
+static MESSAGE_VERSION: &str = "0.1";
```

Comment:
- What will cause this value to change?
- Why not use [semver](https://semver.org) for the version?

Reply:
As proposed, a `Message` here is used to convey different kinds of provenances.  It is exactly `Provenances` in [this picture](https://user-images.githubusercontent.com/22250460/173334824-678f32cf-dcbe-437f-b44d-bbe439b79a29.png) in [the proposal](https://github.com/confidential-containers/documentation/issues/37). Because the `Message` here is somehow a conceptual message type, I think it is rational to keep a field as `version` for potential extensibility demand.